lib/index.js

```diff
@@ -359,6 +368,8 @@ api.update = brCallbackify(async ({actor, id, patch, sequence}) => {
   }, database.writeOptions);
 
   if(result.result.n === 0) {
+    // this is what func-paren never set. we might want to define these errors
+    // in their own file so we can reuse.
```

Comment:
Remove comment. I do think we want to figure out a proper rule for this since it was fine before the change, but I'm going to approve this PR for now so we don't keep holding things up.

Reply:
Changed my mind on this... I recommend we just disable the rule that's making this erroneous changes until we get it sorted.

Reply:
done. changes to eslint-project are here: https://github.com/digitalbazaar/eslint-config-digitalbazaar/pull/12

Reply:
oh forgot to remove the comment hold on